docs/spec/bicep.md

```diff
@@ -62,6 +62,62 @@ var name = 'steve'
 var myVar3 = 'hello ${name}!'
 ```
 
+### Multi-line strings
+> Requires Bicep CLI v0.3 or later
+
+Multi-line strings are defined by a sequence of characters between 3 or more single quote (`'`) characters. Characters that are entered between the opening and closing sequence are read verbatim, and no escaping is necessary or possible.
```

Comment:
>between [](start = 136, length = 7)

The part after "between..." is not entirely true. I think we should mention something like "except the rules below".

Reply:
Good point